lib/ossert/repositories.rb

```diff
@@ -101,7 +101,7 @@ def initialize(stored_project, stats_type)
       end
 
       def coerce_value(value)
-        DateTime.parse(value)
+        Date.parse(value)
```

Comment:
Does it works fine with`Date` instead of `DateTime`?

Reply:
yes, I checked.

Reply:
@sclinede

Reply:
Could you, please, verify it without ActiveSupport?